server/middleware/setUpGovukOneLogin.ts

```diff
@@ -38,11 +40,19 @@ export default function setUpGovukOneLogin(): Router {
       })(req, res, next)
     })
 
-    router.use('/sign-out', (req, res, next) => {
+    router.use(paths.SIGN_OUT, async (req, res, next) => {
       if (req.user) {
+        const idToken = await idTokenStore.getToken(encodeURIComponent(req.user.sub))
         req.logout(err => {
           if (err) return next(err)
-          return req.session.destroy(() => res.redirect(client.endSessionUrl()))
+          return req.session.destroy(() =>
+            res.redirect(
+              client.endSessionUrl({
+                id_token_hint: idToken,
+                post_logout_redirect_uri: `${config.domain}${paths.SIGNED_OUT}`,
+              }),
```

Comment:
The crux of this change: pass the token and logout redirect URI when calling OL's log out.